### ISSUE TITLE
Model selection

### ISSUE THREAD

**Issue by djhocking** [Regional-Fish-Modeling/YOYgrowth#4]
Since AIC and BIC were both used and gave different answers, I'm unclear how the best model was selected for inference (it didn't appear to be either best model from AIC or BIC).

**Comment by esnook** [comment]
Yes, we're not actually sure what to do about this.  One recommendation I found suggested reporting both AIC and BIC.  AIC said that the full model was the best, while BIC essentially chooses the model that is not null, but has the fewest degrees of freedom.  We tested a series of models, but what we're really interested in is which covariates are important. The standardized fixed effects figure and the individual effects figures show the covariates with significant relationships.  If you add in a model with just the significant covariates as fixed effects, AIC still chooses the full model and BIC chooses the significant effects model (I think I still had this labeled as best model in the code, sorry for the confusion). So, we either include the table of model comparisons with or without the significant effects only model, OR we just present results of the full model and discuss the covariates that are significant.  Suggestions?

**Comment by djhocking** [comment]
Since AIC is generally the accepted method in ecology and we are interested in the covariates rather than primarily for parsimonious predictive ability, I would just skip all the BIC stuff and just use AIC. This selects the full model, which is fine, and we just discuss the results of that model. I wouldn't even emphasis significance too much but rather focus on effect size, direction, and ecological/evolutionary/conservation relevance.

**Comment by bletcher** [comment]
I agree with Dan

On Thu, Apr 28, 2016 at 9:48 AM, Daniel J. Hocking <notifications@github.com
> wrote:

> Since AIC is generally the accepted method in ecology and we are
> interested in the covariates rather than primarily for parsimonious
> predictive ability, I would just skip all the BIC stuff and just use AIC.
> This selects the full model, which is fine, and we just discuss the results
> of that model. I wouldn't even emphasis significance too much but rather
> focus on effect size, direction, and ecological/evolutionary/conservation
> relevance.
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/Regional-Fish-Modeling/YOYgrowth/issues/4#issuecomment-215429047>
>



-- 

 Silvio O. Conte Anadromous Fish Research Center, U.S. Geological Survey
 P.O. Box 796 -- One Migratory Way
 Turners Falls, MA 01376
 (413) 863-3803
 Cell: (413) 522-9417
 FAX (413) 863-9810

ben_letcher@usgs.gov
bletcher@eco.umass.edu
http://www.lsc.usgs.gov/?q=cafb-research

**Comment by txw19** [comment]
I also agree with Dan and this approach. However, not only do I think we should not emphasize "statistical significance", I don't think we should report any p-vlaues at all. It's not good practice to mix null hypothesis testing (statistical significance and P-values) and information theoretic approaches in the same manuscript because they are different paradigms. Burnham and Anderson have written about this, but many people still do it. As Dan stated, stick with reporting effect size/direction, uncertainty, and ecological significance.

**Comment by ykanno** [comment]
Me too. Yoichiro

**Comment by esnook** [comment]
OK, sounds good. Thanks all.
